Assets/MixedRealityToolkit.Extensions/SpectatorView/.gitignore

```diff
@@ -0,0 +1,67 @@
+[Ll]ibrary/
+[Tt]emp/
+[Oo]bj/
+[Bb]uild/
+[Bb]uilds/
+[Ll]ogs/
+[Aa]pp/
+
+# Uncomment this line if you wish to ignore the asset store tools plugin
+# [Aa]ssets/AssetStoreTools*

```

Comment:
Why are we adding a second ignore file?

Reply:
It looks like my comments below did not get added to this conversation correctly. I've gone ahead and updated this .gitignore to only check that the various plugin files don't accidentally get checked in.